src/modules/launcher/Wox.Plugin/Common/Win32/NativeMethods.cs

```diff
@@ -493,28 +493,28 @@ public enum DwmNCRenderingPolicies
     public enum DwmWindowAttributes
     {
         NCRenderingEnabled = 1,
-        NCRenderingPolicy,
-        TransitionsForceDisabled,
-        AllowNCPaint,
-        CaptionButtonBounds,
-        NonClientRtlLayout,
-        ForceIconicRepresentation,
-        Flip3DPolicy,
-        ExtendedFrameBounds,
-        HasIconicBitmap,
-        DisallowPeek,
-        ExcludedFromPeek,
-        Cloak,
-        Cloaked,
-        FreezeRepresentation,
-        PassiveUpdateMode,
-        UseHostbackdropbrush,
-        UseImmersiveDarkMode,
-        WindowCornerPreference,
-        BorderColor,
-        CaptionColor,
-        TextColor,
-        VisibleFrameBorderThickness,
+        NCRenderingPolicy = 2,
+        TransitionsForceDisabled = 3,
+        AllowNCPaint = 4,
+        CaptionButtonBounds = 5,
+        NonClientRtlLayout = 6,
+        ForceIconicRepresentation = 7,
+        Flip3DPolicy = 8,
+        ExtendedFrameBounds = 9,
+        HasIconicBitmap = 10,
+        DisallowPeek = 11,
+        ExcludedFromPeek = 12,
+        Cloak = 13,
+        Cloaked = 14,
+        FreezeRepresentation = 15,
+        PassiveUpdateMode = 16,
+        UseHostbackdropbrush = 17,
+        UseImmersiveDarkMode = 20,
+        WindowCornerPreference = 33,
+        BorderColor = 34,
+        CaptionColor = 35,
+        TextColor = 36,
+        VisibleFrameBorderThickness = 37,
         Last,
```

Comment:
`=38`?

Reply:
Let's leave that one without a number, so it's always +1 then the previous one. I think that's there just so we have a "stop" if we want to go in a loop 🤔  That was something that was usual to do in C code.